frontend/server/k8s-helper.ts

```diff
@@ -62,6 +62,62 @@ function getNameOfViewerResource(logdir: string): string {
   return 'viewer-' + crypto.SHA1(logdir);
 }
 
+function parseTensorboardLogDir(logdir: string, podTemplateSpec: object): string {
+  const urls: string[] = [];
+  const seriesParts = logdir.split(',');
+  for (const seriesPart of seriesParts) {
+    const strPath = seriesPart.replace(/Series\d*:/g, '');
```

Comment:
nit: we should be more strict I think
```
seriesPart.replace(/^Series\d+:/g, '');
```
because Series can only be at the beginning and there should be at least one digit